src/lib/bcrypt/index.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable require-jsdoc */
+import bcrypt from 'bcrypt';
+
+class Bcrypt {
+  async hashPwd(plainTextPassword) {
+    return bcrypt.hash(plainTextPassword, 10);
+  }
+
+  async comparePwd(plainTextPassword, hash) {
+    return bcrypt.compare(plainTextPassword, hash);
+  }
```

Comment:
Lets have these as static methods:

```suggestion
  async static hashPwd(plainTextPassword) {
    return bcrypt.hash(plainTextPassword, 10);
  }

  async static comparePwd(plainTextPassword, hash) {
    return bcrypt.compare(plainTextPassword, hash);
  }
```

so we will use this as `Bcrypt.hashPwd('sdfs')`